### PR TITLE
docs: Add LICENSE and update package.json files

### PR DESCRIPTION
**Pull request by pmareke**
EX-3932

### CODE REVIEW

File: packages/jest-utils/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# x-deep-merge
+
+Utility for deep cloning objects.
```
Comment (LuisMartinez15) [inline packages/jest-utils/README.md:3]:
```suggestion
# x-jest-utils

Jest extensions for easier testing.
```

File: packages/jest-utils/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+### How to install
+
+```
+npm install @empathyco/x-deep-merge --save
```
Comment (LuisMartinez15) [inline packages/jest-utils/README.md:8]:
```suggestion
npm install @empathyco/x-jest-utils --save-dev
```

File: packages/jest-utils/package.json [hunk 3]
```diff
@@ -2,6 +2,9 @@
   "name": "@empathyco/x-jest-utils",
```
Comment (LuisMartinez15) [inline packages/jest-utils/package.json:2]: If you don't mind, can you follow the same structure as in the other packages?

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]

File: packages/jest-utils/package.json (hunk 3 above)
Comment (pmareke) [reply packages/jest-utils/package.json:2]: Sure!

**Review by LuisMartinez15 — APPROVED** [review]

**Comment by javieri-empathy** [comment]
Run `npm ci` from the root folder to update the `package-lock.json`

File: packages/jest-utils/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# x-jest-utils
```
Comment (javieri-empathy) [inline packages/jest-utils/README.md:1]: I would add a "how to use" section:

```
# x-jest-utils

Jest extensions for easier testing.


## How to install

```
npm install @empathyco/x-jest-utils --save-dev
```

## How to use

To register the extended matchers, just import the package in your tests. The recommended way to do so is
using the [setupFilesAfterEnv](https://jestjs.io/docs/configuration#setupfilesafterenv-array) option from your
Jest config file.

```js
// File jest.config.js
module.exports = {
  setupFilesAfterEnv: ['./tests.setup.js'],
  testMatch: ['<rootDir>/**/*.spec.js']
};
```

```js
// File tests.setup.js
import '@empathyco/x-jest-utils';
```

If you want to find out all the matchers, check the [src/jest-utils.types.ts](src/jest-utils.types.ts) file.

## How to update the version

You can check if a new version has been published running [npm outdated](https://docs.npmjs.com/cli/outdated):

`npm outdated @empathyco/x-jest-utils`

And update it using [npm update](https://docs.npmjs.com/cli/update):

`npm update --save @empathyco/x-jest-utils`

```

File: packages/jest-utils/README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+### How to update the version
+
+You can check if a new version has been published running [npm outdated](https://docs.npmjs.com/cli/outdated):
+
+`npm outdated @empathyco/x-jest-utils`
+
+And update it using [npm update](https://docs.npmjs.com/cli/update):
+
+`npm update --save @empathyco/x-jest-utils`
```
Comment (javieri-empathy) [inline packages/jest-utils/README.md:21]: I would add a "how to use" section:

```suggestion
# x-jest-utils

Jest extensions for easier testing.


## How to install

\```
npm install @empathyco/x-jest-utils --save-dev
\```

## How to use

To register the extended matchers, just import the package in your tests. The recommended way to do so is
using the [setupFilesAfterEnv](https://jestjs.io/docs/configuration#setupfilesafterenv-array) option from your
Jest config file.

\```js
// File jest.config.js
module.exports = {
  setupFilesAfterEnv: ['./tests.setup.js'],
  testMatch: ['<rootDir>/**/*.spec.js']
};
\```

\```js
// File tests.setup.js
import '@empathyco/x-jest-utils';
\```

If you want to find out all the matchers, check the [src/jest-utils.types.ts](src/jest-utils.types.ts) file.

## How to update the version

You can check if a new version has been published running [npm outdated](https://docs.npmjs.com/cli/outdated):

`npm outdated @empathyco/x-jest-utils`

And update it using [npm update](https://docs.npmjs.com/cli/update):

`npm update --save @empathyco/x-jest-utils`

```

Comment (javieri-empathy) [reply packages/jest-utils/README.md:21]: Be careful because I don't know how to tell github to escape the fucking backticks inside the proposed code

Comment (pmareke) [reply packages/jest-utils/README.md]: Done.

**Comment by pmareke** [comment]
> Run `npm ci` from the root folder to update the `package-lock.json`

I did a couple of times but the `package-lock.json` doesn't change.

**Review by javieri-empathy — APPROVED** [review]

**Review by LuisMartinez15 — APPROVED** [review]

**Pull request merged**